pkg/virt-controller/services/template.go

```diff
@@ -533,6 +554,9 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		},
 		Resources: resources,
 		Ports:     ports,
+		Env: []k8sv1.EnvVar{
```

Comment:
So the idea here is that virt-controller looks up the space toleration, because it has access to cluster resources (e.g. ConfigMaps), but passes this value down the chain as an Env var because the volume hasn't been create yet thus can't be verified?

Reply:
yes, only virt-launcher has the volumes mounted and can verify the available space, when creating  disk images on empty PVs, but it can't look up the configuration.